### PR TITLE
Plumb the configured gsuite domain down into the profile creation page

### PR DESCRIPTION
**Pull request by calbach**

### CODE REVIEW

File: ui/src/app/views/account-creation/component.html [hunk 1]
```diff
@@ -13,7 +13,7 @@ <h2>Create your account</h2>
     </div>
     <div class="form-section">
       <input type="text" class="short-input" id="username" name="username" [(ngModel)]="profile.username" (ngModelChange)="usernameChanged()" placeholder="Create Account">
-      <h5 style="display: inline-block;">@fake-research-aou.org</h5>
+      <h5 style="display: inline-block;">@{{ gsuiteDomain }}</h5>
```
Comment (blrubenstein) [inline ui/src/app/views/account-creation/component.html]: Nit: Spaces after and before `gsuiteDomain`

Comment (calbach) [reply ui/src/app/views/account-creation/component.html]: Done

**Comment by calbach** [comment]
Tests fixed, had to add a stub for this service as well.

**Comment by calbach** [comment]
Interesting, prepush lint hook seemed not to run when I did a force push, will look into that later. The API failure is a known issue with https://github.com/all-of-us/workbench/pull/660 out to fix.

**Pull request merged**